### DISCUSSION TITLE
📚 Documentation: What's the realtime websocket URL?

### DISCUSSION THREAD

**Discussion by anwar-28** [appwrite/appwrite#8118] [Q&A]
### 💭 Description

i need help on my websocket url

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by SafakB** [comment]
Hey, I think that's what you're talking about. 

```
wss://<APPWRITE_ENDPOINT>/v1/realtime?<PARAMETERS>
```

```
wss://app.example.com/v1/realtime?project=6640dcdd0038291cd097&channels%5B%5D=databases.6640dcf00018f6322ff1.collections.663622d7000907ffeb72.documents
```

I use Socket operations during game development with Unity.
